### PR TITLE
Update README and add python requirements

### PR DESCRIPTION
**Pull request by patbro**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+PyJWT==2.0.1
+requests==2.25.1
```
Comment (mathijsdelangen) [inline requirements.txt:2]: Add argparse requirement

**Review by mathijsdelangen — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,15 +1,22 @@
 # AutoTenez
 Automates the process of making tennis court reservations for you.
 
+## Setup
+1. Install all necessary pip dependencies using `python -m pip install -r requirements.txt`.
+2. Open `AutoTenez.py`. Enter your email address, password, and set `only_retrieve_your_external_reference` to `True`.
+3. Run `python AutoTenez.py -t 00:00 -f 0` to retrieve your own external reference.
+4. Share your external reference with a friend, and set `only_retrieve_your_external_reference` to `False`.
```
Comment (mathijsdelangen) [inline README.md]:
```suggestion
1. Install all necessary pip dependencies using `python -m pip install -r requirements.txt`.
1. Open `AutoTenez.py`. Enter your email address, password, and set `only_retrieve_your_external_reference` to `True`.
1. Run `python AutoTenez.py -t 00:00 -f 0` to retrieve your own external reference.
1. Share your external reference with a friend, and set `only_retrieve_your_external_reference` to `False`.
```

Comment (mathijsdelangen) [reply README.md]: - Markdown will fix the enumeration.

File: requirements.txt (hunk 1 above)
Comment (patbro) [reply requirements.txt:2]: Argparse is part of the standard library, hence it's not necessary to add argparse as a requirement (that's also why it's not added automagically after running `pip freeze`).

**Review by mathijsdelangen — APPROVED** [review]

**Pull request merged**